test/customer_repository_test.rb

```diff
@@ -0,0 +1,37 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/customer_repository'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.